spec/docs/users_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "swagger_helper"
+
+describe "API", type: :request do
+  path "/api/v1/users" do
```

Comment:
This needs to just be `/users` I think? It's generating `curl -X GET "https://ecf-review-pr-314.london.cloudapps.digital/api/v1/api/v1/users" -H "accept: application/json"`